src/test/README.md

```diff
@@ -74,3 +74,19 @@ start debugging, just like you would with any other program:
 ```bash
 gdb src/test/test_bitcoin
```

Comment:
On a second thought. Won't this be enough to observe segmentation faults in gdb if they appear? Is there a reason to take the extra step of a core dump?

Reply:
Yeah, this is a good point; willfix.